libs/langchain/langchain/schema/runnable/config.py

```diff
@@ -25,3 +26,42 @@ class RunnableConfig(TypedDict, total=False):
     Callbacks for this call and any sub-calls (eg. a Chain calling an LLM).
     Tags are passed to all callbacks, metadata is passed to handle*Start callbacks.
     """
+
+    _locals: Dict[str, Any]
```

Comment:
Seeing as this is a dict key, let's just rename to `locals` ?

Reply:
To avoid shadowing builtins  `_` should be used as a suffix rather than a prefix.

I'm OK with either `locals` or `locals_`, but there are alternatives like `local_vars`

Reply:
wanted to suggest it was private and shouldn't be directly accessed, since you can do stuff like `config._locals` with typed dict